src/docs/product/accounts/quotas/manage-event-stream-guide.mdx

```diff
@@ -111,60 +111,74 @@ To review the error events dropped because of spike protection, go to the "Usage
 
-### New Heuristic Changes
+### New Spike Protection Calculations
 
 <Include name="limited-avail-note.mdx" />
 
-Spike protection is enabled for every project by default, and when it's enabled, Sentry continually monitors for spikes. You can confirm that it's enabled in **Settings > Projects > _Select Project_ > General Settings**.
+Limited availability spike protection is a project-level tool that helps prevent quota overconsumption. It's enabled for every project by default, and when it's enabled, Sentry continually monitors for spikes. You can confirm that it's enabled in **[Project] > Settings > General Settings**.
 
-The way our spike protection algorithm essentially works is by using a weighted average of your events over the past 168 hours (past 7 days), applying a multiplier to that number, comparing this final number against a floor bound that is determined using your quota, and setting that as your spike limit.
+Our spike protection algorithm does the following:
 
-#### Spike Protection Inputs
+- Uses a weighted average of your events over the past 168 hours (seven days)
+- Applies a multiplier to that number
+- Compares this final number against a minimum number of events to trigger a spike that is determined using your quota
+- Sets this as your spike limit
 
-- Number of projects
-- Quota (per event type)
-- Events in the past 7 days
+#### Setting the Spike Limit
 
-#### Floor Bound Calculation
+There are two ways that we can set your spike limit, or the number of events that trigger a spike:
 
-To break it down even further, the first step of this algorithm identifies a floor bound that is calculated using your quota. This bound takes the max of either 500 events or (3 \* your quota)/(720 \* number of projects) - the latter number represents your project using up 3 times your overall quota in 30 days if events are continually ingested at this hourly rate, thus flagging for a potential spike.
+- [Minimum Event Calculation](#minimum-event-calculation) - A calculation that determines a minimum number of events
+- [Usage-Based Calculation](#usage-based-calculation) - A projection based on your past usage
 
-#### Spike Limit Calculation
+The spike limit for each hour is set to either the minimum event or usage-based calculation — whichever is higher. This is done for a number of reasons. Firstly, using a minimum event calculation protects smaller or new projects. New projects that don't have a week’s worth of data to use to calibrate spike limits can use this minimum number of events, an adaptation of the organization’s quota, to approximate appropriate limits. Additionally, this calculation can be used to minimize false positives in smaller or new projects so that spikes aren’t flagged incorrectly.
```

Comment:
```suggestion
The spike limit for each hour is set using either the minimum event or usage-based calculation — whichever is higher. This is done for a number of reasons. Firstly, using a minimum event calculation protects smaller or new projects. New projects that don't have a week’s worth of data to use to calibrate spike limits can use this minimum number of events, an adaptation of the organization’s quota, to approximate appropriate limits. Additionally, this calculation can be used to minimize false positives in smaller or new projects so that spikes aren’t flagged incorrectly.
```